snuba/query/snql/discover_entity_selection.py

```diff
@@ -32,9 +41,81 @@
 
 """
 
-EVENTS = EntityKey.DISCOVER_EVENTS
-TRANSACTIONS = EntityKey.DISCOVER_TRANSACTIONS
-EVENTS_AND_TRANSACTIONS = EntityKey.DISCOVER
+
+# NOTE: These will need to be updated if/when events and transaction columns change.
```

Comment:
can't we just get this from the entities?

Reply:
Yes but I think that should be in a separate PR, which I'm working on.